easybuild/easyconfigs/r/RASPA2/RASPA2-2.0.3-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'RASPA2'
+version = '2.0.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/numat/RASPA2'
+description = """A general purpose classical simulation package that can be used for the simulation of
+ molecules in gases, fluids, zeolites, aluminosilicates,
+ metal-organic frameworks, carbon nanotubes and external fields."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['https://pypi.python.org/packages/source/r/RASPA2/']
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.12'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/raspa-dir', 'bin/simulate'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(name)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg']
```

Comment:
@hajgato since this `.egg` directory may also become a (zipped) `.egg` file under some circumstances, we typically rely on just checking for `lib/python%(pyshortver)s/site-packages` + the `import` check that is done by `PythonPackage`...